### PR TITLE
Use embed format for OKIB message

### PR DESCRIPTION
**Pull request by jmrico01**
#### Original
ibl2

#### Proposed change
ibl

The big icons are nice, but the gatherer at the bottom is a bit sketchy. It's also more vertical space... I am OK with whatever you prefer @Archimonde666

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -525,8 +525,6 @@ async def on_message(message):
 
 async def gather():
     gather_list_string = " ".join([member.mention for member in _okib_members])
-    # TODO combine these? can't combine the message sends, but can combine the ensure_display
-    # you doing it wront => the purpose of making a encapsulating function is to actually ensuredisplay the whole thing not everything inside it ^^
```
Comment (jmrico01) [inline main.py:545]: @Archimonde666 I see where I went wrong now

Comment (Archimonde666) [reply main.py:545]: This is quite a nice way to code I guess :P

**Review by Archimonde666 — APPROVED** [review]

**Review by Archimonde666 — CHANGES_REQUESTED** [review]
I agree that the « gatherer asking » should be at the top dunno if possible but you could just go with the OKIB GATHER STRING At the end after the embed thing

**Comment by jmrico01** [comment]
> 
> 
> I agree that the « gatherer asking » should be at the top dunno if possible but you could just go with the OKIB GATHER STRING At the end after the embed thing

I just tried this but it's not possible to flip the order. We'll have to either change the `gatherer asks:` line or just accept the message as it is without embed.

**Review by Archimonde666 — DISMISSED** [review]

File: main.py [hunk 2]
```diff
@@ -559,16 +557,16 @@ async def combinator3000(*args):
...
-        OKIB_EMOJI_STRING, len(_okib_members), OKIB_GATHER_PLAYERS, okib_list_string,
-        NOIB_EMOJI_STRING, noib_list_string
-    )
-    #await ensure_display((_okib_channel.fetch_message(_okib_message_id)).edit, content=_list_content)
+    _list_content_embed = discord.Embed(
+        description="{} {}/{} : {}\n{} : {}".format(
+            OKIB_EMOJI_STRING, len(_okib_members), OKIB_GATHER_PLAYERS, okib_list_string,
+            NOIB_EMOJI_STRING, noib_list_string
+        )
+    ).set_footer(text="(started by {})".format(_gatherer.display_name))
```
Comment (jmrico01) [inline main.py:569]: `started by`? `asked by`? `Gatherer: `? `called by`?

**Comment by Archimonde666** [comment]
I don't like it at all. First, why Embed ? it looks perfect as it is !
refering to what was said in issue 16 : [](https://github.com/impossible-bosses/ibce-bots/issues/16)
the goal is still to keep it as compact as possible so as long as embed is that wide why ?

**Comment by jmrico01** [comment]
fair enough, byebye then!

**Comment by Archimonde666** [comment]
Therefore i'm interested in the bigger icons, did it solves it with only one returned value or did you coded something more complex ?

**Comment by jmrico01** [comment]
The bigger icons are possible because everything apart from the emojis is inside the embed. The message text itself only contains emojis, so they get enlarged. This means we'll always need a new line for the gatherer inside the embed, that's why it's so tall.

**Pull request closed without merging**